contrib/kubespray/quick-start-kubespray.sh

```diff
@@ -38,5 +38,8 @@ ansible-playbook -i ${HOME}/pai-pre-check/pre-check.yml set-host-daemon-port-ran
 echo "Performing pre-installation..."
 ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml pre-installation.yml || exit $?
 
+echo "Performing docker-cache config distribution..."
+ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml docker-cache-config-distribute.yml || exit $?
+
```

Comment:
Does this impose any additional software requirements [here](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-guide.html#installation-requirements) ?

Reply:
It is a python3 script to add specific registry-mirror and insecure-registry. Do we need add python3 as a prerequisite ? The script is at contrib/kubespray/roles/docker-cache/install/files/

Reply:
> It is a python3 script to add specific registry-mirror and insecure-registry. Do we need add python3 as a prerequisite ? The script is at contrib/kubespray/roles/docker-cache/install/files/

python3 will be installed in `contrib/kubespray/script/environment.sh`